### PR TITLE
Issue 45 dockerize react UI

### PR DESCRIPTION
**Pull request by Wolfinbarger**
Created dockerfile for frontend and compose file in root directory. One issue is that if URL contains localhost there is a chance it will not pull data until the localhost is replaced with the IP instead. Will fix this after migration to fastAPI, if it persists there.

To run simply be in the root directory in terminal, ensure the compose file is there, and enter 'docker compose up' to exit run 'docker compose down'.

### CODE REVIEW

File: src/app/compose.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+services:
+  frontend:
+    image: frontend/game-recommender:latest
```
Comment (FreakyNobleGas) [inline src/app/compose.yaml:3]: Can we add this to our Dockerhub?

Comment (Wolfinbarger) [reply src/app/compose.yaml:3]: Did you add me to it as a collaborator?

Comment (Wolfinbarger) [reply src/app/compose.yaml]: Changed line 3 to freakynoblegas/game-recommender:frontend

**Review by FreakyNobleGas — APPROVED** [review]
Nice job! Sorry for the huge delay in reviewing this. I think we might run into some issues with the docker-compose.yaml file pointing to the wrong API image, but we can fix that in a follow-up PR if you want.

File: src/app/src/components/cards/Cards.jsx
Comment (FreakyNobleGas) [inline src/app/src/components/cards/Cards.jsx:1]: I would recommend removing these white space changes unless they came from a linter. It's makes it easier to investigate issues if you need to dig into the file history.

**Pull request merged**